metadata/email_addressesMetaData.php

```diff
@@ -80,13 +80,23 @@
             'default'        => 0,
             'vname'         => 'LBL_OPT_OUT',
         ),
+        'opt_in_email_created' => array(
+            'name' => 'opt_in_email_created',
+            'type' => 'datetime',
+            'vname' => 'LBL_OPT_IN_EMAIL_CREATED',
+        ),
         'confirm_opt_in' => array(
             'name'            => 'confirm_opt_in',
             'type'            => 'bool',
             'default'        => 0,
             'vname'         => 'LBL_CONFIRM_OPT_IN',
         ),
-            
+
+        'confirmed_opt_in_date' => array(
```

Comment:
So there are 3 fields which are related to this topic. 

confirmed_op_in - is just to make it easy to check if an email address is opted in.

opt_in_email_created - records when the last opt in email was created (and maybe sent)

confirmed_opt_in_date meet the GDPR requirement that you have to store the date when the user confirm to opt in. This would be used to create a report.